lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystemConfig.java

```diff
@@ -184,6 +197,20 @@ public S3FileSystemConfig setStsRegion(String stsRegion)
         return this;
     }
 
+    @NotNull
+    public ObjectCannedAcl getCannedAcl()
+    {
+        return objectCannedAcl;
+    }
+
+    @Config("s3.canned-acl")
+    @ConfigDescription("Canned ACL (predefined grants) to manage access to buckets and objects")
```

Comment:
Remove "buckets" since in this context it's only relevant for objects (canned ACLs can also be set at the bucket level but we're not doing that here)

> Canned ACL (predefined grants) to manage access to objects

Reply:
Sorry, looks like this one was not updated

Reply:
Sorry . missed it. updated now